src/translations/screens/TripSearch.ts

```diff
@@ -202,15 +202,12 @@ const TripSearchTexts = {
         'Nå kan du bruke filter i reisesøk og få reiseforslag tilpasset dine behov.',
         'You can now use filters in travel searches and get travel suggestions adapted to your needs.',
       ),
-      part2: _(
-        'Velg transportmiddel, hvor lang tid du ønsker å ha mellom bytter og hvor raskt du beveger deg.',
-        'Choose your means of transport, how long time you prefer between changes and how fast you walk.',
-      ),
+      part2: _('Velg transportmiddel.', 'Choose your means of transport.'),
```

Comment:
I understand this is temporary, but can we try to make this sound a little bit more inviting? 😆

Something like

```suggestion
      part2: _('Velg hvilke transportmiddel du ønsker å reise med.', 'Select the means of transport you want to use for your trip.'),
```